src/main/java/pl/sknikod/kodemy/configuration/AppConfig.java

```diff
@@ -39,9 +40,9 @@ public static class SecurityRoleProperties {
         private String defaultRole;
         private Map<String, Set<String>> privileges = new LinkedHashMap<>();
 
-        public Set<SimpleGrantedAuthority> getPrivileges(String role) {
+        public Set<SimpleGrantedAuthority> getPrivileges(RoleName role) {
             return Option
-                    .of(privileges.get(role))
+                    .of(privileges.get(role.toString()))
```

Comment:
Sprawdzałeś pozostały kod, który tą metode wykorzystuje? Trzeba to sprawdzić by nie zakłucać procesów. Jeśli działa to może być.

Reply:
Z tego co mi się wydaje to tak, są tylko dwie i obsługują OAuth2 a logowałem się i działa

Reply:
A rejestrację próbowałeś?